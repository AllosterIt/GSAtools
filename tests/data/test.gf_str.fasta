>Time    0.000
QNDEGAAQLTITUUWUUUUUWUUUPRFGADMLNFTUUUWUUTXNADHNAGAGHRRHSPUKSTVVTWUUUUXMWQOEAAOAJPVUTVWUUXVRLXIRRLDFJHVQHJUUUUUUUUUUVWVTV
>Time    1.000
MNCBDAMQLTCVVXVUUUUWUUUUPQFDMAALNOVUUUUWUVXNDDONBAAGHRRHSPUKSTVTTUUUWUXMXQJEGAFEIPUUTUVWUUWRCXFRRCGFKHUEKJVUWUUUVUUUUUUTV
>Time    2.000
QNDEGAAQLTITUUWUUWUUUVUUKQCBGCMLNFTUUUUWUVXNDDONBABDFSRPSKUKSSVTVWUUUUXMWQOEMAFEHKUUTUVWUUWRCWISRAAIKHVEKJVUUUUUUUWVUUUTT
>Time    3.000
QNDBGAMQLTGTUUUWUUUUUUUUKQIBGIBMNOVUWVUUUVXNGDKNMADGHRRHSPUKSTVTTUUUWUXMXQOEMAHBJPVUTVWUUWVRLXIRRGDIOHVEOJVUWUUUVUUUUUUTV
>Time    4.000
QNGEDAMQLTCTUUUUUUXUWUUUPRDEDIMLNOVUUUUWUTXNDCKNAAEDFSRHSPUPRSVTUUWUUUXMWRIBLGFEOPVUTVUUXUWRCWFRRLCFFHUEPJUVUUUUUUUUVXVTV
>Time    5.000
QNDEAALELNFTWUUUUUWUUUUUKQIEAFMMNFTUUUWUUTXNADHNGLBDFSRHSPUPSSVSUUUXVWXMWQOEMAFBKPVUTVUUXVNQFWIRRGDIOHUQHJVUUUUUUUUUUUWSV
>Time    6.000
QNCBAALELNFTWUUUUUWUUUUUKQIEAFMMNFTUUUWUUTXNADHNGLBDFSRHSPUPSSVSUUUXVWXMWQOEMAFBKPVUTVUUXUWRCWISRACJOHVQFOVUWUUUVUUUUUUTV
>Time    7.000
QNDBGAMQLTGTUUUWUUUUUUUUKQIBGIBMNOVUWVUUUVXNCCONBCBBHSROSPUKTSVSVUUWVWXMNSHAGDOEIKUUTVUUXVNQFXFRSLIFHPWEKJVUUUUUUUUUUXUTT
>Time    8.000
QNCBAALELNFTWUUUUUWUUUUUKQIEAFMMNFTUUUWUUTXNADHNGLBDFSRHSPUPSSVSUUUXVWXMWQOEMAFBKPVUTVUUXUWRCWISRACOKHUEKJVUUUUUUUWVUUUTT
>Time    9.000
MNCBDAMQLTCVVXVUUUUWUUUUPQIEAFAMNOTUUUXUUTXNADHNGLBDFSRHSPUPSSVSUUUXVWWMWQJEADOECPUUVTXVUXVRLXISQMMIPHVQHJVUUUUUVUUUUXVTT
>Time   10.000
QNDEGAAQLTITUUWUUUUUWUUUPRIAGFAMNJVUUVUWUVXNGGHNACDGFRRHSPUPSSUSVUUXVWWMXQFELDFEJPUUVVUVXVWSLUCRRLDFFHUEKJUVUVUUUUUUUXUTT
