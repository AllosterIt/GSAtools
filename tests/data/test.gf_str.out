QNDEGAAQLTITUUWUUUUUWUUUPRFGADMLNFTUUUWUUTXNADHNAGAGHRRHSPUKSTVVTWUUUUXMWQOEAAOAJPVUTVWUUXVRLXIRRLDFJHVQHJUUUUUUUUUUVWVTV
MNCBDAMQLTCVVXVUUUUWUUUUPQFDMAALNOVUUUUWUVXNDDONBAAGHRRHSPUKSTVTTUUUWUXMXQJEGAFEIPUUTUVWUUWRCXFRRCGFKHUEKJVUWUUUVUUUUUUTV
QNDEGAAQLTITUUWUUWUUUVUUKQCBGCMLNFTUUUUWUVXNDDONBABDFSRPSKUKSSVTVWUUUUXMWQOEMAFEHKUUTUVWUUWRCWISRAAIKHVEKJVUUUUUUUWVUUUTT
QNDBGAMQLTGTUUUWUUUUUUUUKQIBGIBMNOVUWVUUUVXNGDKNMADGHRRHSPUKSTVTTUUUWUXMXQOEMAHBJPVUTVWUUWVRLXIRRGDIOHVEOJVUWUUUVUUUUUUTV
QNGEDAMQLTCTUUUUUUXUWUUUPRDEDIMLNOVUUUUWUTXNDCKNAAEDFSRHSPUPRSVTUUWUUUXMWRIBLGFEOPVUTVUUXUWRCWFRRLCFFHUEPJUVUUUUUUUUVXVTV
QNDEAALELNFTWUUUUUWUUUUUKQIEAFMMNFTUUUWUUTXNADHNGLBDFSRHSPUPSSVSUUUXVWXMWQOEMAFBKPVUTVUUXVNQFWIRRGDIOHUQHJVUUUUUUUUUUUWSV
QNCBAALELNFTWUUUUUWUUUUUKQIEAFMMNFTUUUWUUTXNADHNGLBDFSRHSPUPSSVSUUUXVWXMWQOEMAFBKPVUTVUUXUWRCWISRACJOHVQFOVUWUUUVUUUUUUTV
QNDBGAMQLTGTUUUWUUUUUUUUKQIBGIBMNOVUWVUUUVXNCCONBCBBHSROSPUKTSVSVUUWVWXMNSHAGDOEIKUUTVUUXVNQFXFRSLIFHPWEKJVUUUUUUUUUUXUTT
QNCBAALELNFTWUUUUUWUUUUUKQIEAFMMNFTUUUWUUTXNADHNGLBDFSRHSPUPSSVSUUUXVWXMWQOEMAFBKPVUTVUUXUWRCWISRACOKHUEKJVUUUUUUUWVUUUTT
MNCBDAMQLTCVVXVUUUUWUUUUPQIEAFAMNOTUUUXUUTXNADHNGLBDFSRHSPUPSSVSUUUXVWWMWQJEADOECPUUVTXVUXVRLXISQMMIPHVQHJVUUUUUVUUUUXVTT
QNDEGAAQLTITUUWUUUUUWUUUPRIAGFAMNJVUUVUWUVXNGGHNACDGFRRHSPUPSSUSVUUXVWWMXQFELDFEJPUUVVUVXVWSLUCRRLDFFHUEKJUVUVUUUUUUUXUTT
